parquet/src/main/java/org/apache/iceberg/parquet/ParquetMetricsRowGroupFilter.java

```diff
@@ -565,6 +565,7 @@ private static boolean mayContainNull(Statistics statistics) {
     return !statistics.isNumNullsSet() || statistics.getNumNulls() > 0;
   }
 
+  @SuppressWarnings("UnusedMethod")
```

Comment:
If this is really unused shouldn't we be removing it? Is it needed for something via reflection?

Reply:
not sure via reflection, so revert